pages/Documents/MessagingChannels/PushNotificationService/overview.md

```diff
@@ -19,16 +19,7 @@ indicator: messaging
 
 Mobile App Messaging provides brands with the ability to provide their customers with an ongoing, continuous conversation that takes place on their time.  For Mobile App Messaging to be successful, push notifications must be configured in order to send customers incoming messages when they are not inside the app.
 
-![PushServices](img/pushservices.png)
+<img src="/img/pushservices.png" alt="Push Notification Sample">
 
-LivePerson’s push service interfaces directly with Apple’s APNS and Google’s FCM push notification services. For brands who wish to refrain from sharing their tokens/certificates, LivePerson offers the option to configure their push to the URIs of their choice and to utilize their own logic.
-
-### Getting Started
-
-1. **Retrieve your domain**. Use the [LivePerson Domain API](agent-domain-domain-api.html) to retrieve this information by providing the following service name:
-
-	* pusher
-
-3. Note the [API terms of use](https://www.liveperson.com/policies/apitou).
-
-4. When using this API, it is recommended that you implement our [Retry Policy and KeepAlive best practices](guides-retry-policy.html).
+LivePerson’s push service interfaces directly with Apple’s APNS and Google’s FCM push notification services. For brands who wish to refrain from sharing their tokens/certificates, LivePerson offers the option to configure their push to the URIs of their choice and to utilize their own logic, see:  
+[Push Proxy](push-notification-service-configuration-push-proxy.html)
```

Comment:
Please add an extra line break between paragraph and image (i.e., there should be one blank line between the two).

Reply:
added